src/unwrap.rs

```diff
@@ -0,0 +1,87 @@
+use crate::utils::{AttrParams, DeriveType, State};
+use convert_case::{Case, Casing};
+use proc_macro2::TokenStream;
+use quote::{format_ident, quote};
+use syn::{DeriveInput, Fields, Ident, Result};
+
+pub fn expand(input: &DeriveInput, trait_name: &'static str) -> Result<TokenStream> {
+    let state = State::with_attr_params(
+        input,
+        trait_name,
+        quote!(),
+        String::from("unwrap_variant"),
```

Comment:
Should be `unwrap` to fix `#[unwrap(ignore)]`.

Reply:
That works, thank you!